Model/Api/Client.php

```diff
@@ -683,7 +683,8 @@ public function getSessionInfo($sessionId, $scopeCode = null)
     {
         $this->scope = $scopeCode;
         $endpoint = $this->getCheckoutApiUri(sprintf('sessions/%s', $sessionId));
-        $request = $this->initRequest($endpoint, $this->getToken())->setBody(null);
+        $request = $this->initRequest($endpoint, $this->getToken())->setBody(null)
```

Comment:
Do you need `setBody(null)`? It's not used in `getTransaction`.

Reply:
@andersem , it is not needed, I've removed it in the next commit.